### ISSUE TITLE
Running a Docker container as a non-root user

### ISSUE THREAD

**Issue by llaville** [miniscruff/changie#397]
**Description**
Docker Run Container in root mode

**Reproduction Steps**
1. Follow instruction at https://github.com/miniscruff/changie/blob/v1.10.0/docs/content/guide/installation/index.md#docker
1. Replace `latest` command by `init`
1. Then run docker command
1. See files properties

**What happened**
Files generated are own by `root` user.

**Expected behavior**
Files must be generated with current user account.

**Additional context**
I recommend to add a note into the [Docker Installation Guide](https://github.com/miniscruff/changie/blob/main/docs/content/guide/installation/index.md#docker), or simply change current syntax with `--user $(id -u):$(id -g) ` Docker run option.

From, i.e
```
docker run --rm --mount type=bind,source=$PWD,target=/src -w /src  ghcr.io/miniscruff/changie:v1.10.0 init
```
To
```
docker run --rm --user $(id -u):$(id -g) --mount type=bind,source=$PWD,target=/src -w /src  ghcr.io/miniscruff/changie:v1.10.0 init
```

TLDR; at https://medium.com/redbubble/running-a-docker-container-as-a-non-root-user-7d2e00f8ee15

**Comment by miniscruff** [comment]
That is a good point, are you able to open a pull request for the doc changes? I can manually build a new site later on.

**Comment by llaville** [comment]
I didn't though you are agree to [combine both issues fixes](https://github.com/miniscruff/changie/pull/401#discussion_r1020528681), otherwise I would have done it already !
